test/unit/config/config-server-side.test.js

```diff
@@ -161,6 +161,15 @@ tap.test('when receiving server-side configuration', (t) => {
     t.end()
   })
 
+  t.test('should disable ai monitoring', (t) => {
+    // Default is `false`, but let's verify it anyway.
+    t.equal(config.ai_monitoring.enabled, false)
```

Comment:
shouldn't this be set to `true`?

Reply:
The default is `false` https://github.com/newrelic/node-newrelic/blob/7cd6aef7ecc32d8647f0313b36be368bfa14a0e9/lib/config/default.js#L1360

Reply:
yea i know but you're basically setting it to false and then getting false back from connect and asserting ai_monitoring.enabled is still false, I don't see what the test is actually asserting